### PR TITLE
task done

### PR DESCRIPTION
**Pull request by Kateryna-Takhtai**
top basic task is done.

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = new Machine[]{truck,bulldozer,excavator};
```
Comment (kateryna-deshchinska) [inline src/main/java/core/basesyntax/MainApp.java:8]:
```suggestion
        Machine[] machines = new Machine[]{truck, bulldozer, excavator};
```

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer does work");
```
Comment (kateryna-deshchinska) [inline src/main/java/core/basesyntax/Bulldozer.java:6]:
```suggestion
        System.out.println("Bulldozer start working);
```

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer does not work");
```
Comment (kateryna-deshchinska) [inline src/main/java/core/basesyntax/Bulldozer.java:11]:
```suggestion
        System.out.println("Bulldozer finished working");
```

Comment (kateryna-deshchinska) [inline src/main/java/core/basesyntax/Bulldozer.java:11]: Make message for different machine more similar, you may use my example below

File: src/main/java/core/basesyntax/MainApp.java [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+        for (Machine machine: machines) {
+            machine.doWork();
+        }
+        for (Machine machine: machines) {
+            machine.stopWork();
+        }
+
```
Comment (kateryna-deshchinska) [inline src/main/java/core/basesyntax/MainApp.java:15]:
```suggestion
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 5]
```diff
@@ -1,5 +1,18 @@
...
+    }
```
Comment (kateryna-deshchinska) [inline src/main/java/core/basesyntax/MainApp.java:17]:
```suggestion
```

**Review by kateryna-deshchinska — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/MainApp.java (hunk 4 above)
Comment (Rommelua) [reply src/main/java/core/basesyntax/MainApp.java:15]: not fixed...

**Review by Rommelua — APPROVED** [review]
I approve, but do not add redundant empty lines and fix **all** mentor's comments before asking new review.

**Pull request closed without merging**